src/core/sys/posix/fcntl.d

```diff
@@ -844,6 +844,126 @@ else version( CRuntime_Musl )
     enum FD_CLOEXEC     = 1;
     int open(in char*, int, ...);
 }
+else version( CRuntime_UClibc )
+{
+    enum F_DUPFD        = 0;
+    enum F_GETFD        = 1;
+    enum F_SETFD        = 2;
+    enum F_GETFL        = 3;
+    enum F_SETFL        = 4;
+
+    version(X86_64)
+    {
+        enum F_GETLK        = 5;
+        enum F_SETLK        = 6;
+        enum F_SETLKW       = 7;
+    }
+    else static if (__USE_FILE_OFFSET64)
```

Comment:
Didn't think you could mix `version` and `static if` like this, does it actually work?

Reply:
Yes, it works. It is also used on the Glibc version, see line 102